src/lib/replication/log_store/home_raft_log_store.cpp

```diff
@@ -293,7 +295,13 @@ bool HomeRaftLogStore::compact(ulong compact_lsn) {
     // we rely on resrouce mgr timer to trigger truncate for all log stores in system;
     // this will be friendly for multiple logstore on same logdev;
 
-    // m_log_store->truncate(to_store_lsn(compact_lsn));
+#ifdef _PRERELEASE
+    if (iomgr_flip::instance()->test_flip("force_home_raft_log_truncate")) {
```

Comment:
Suggest a parameterized flip, so you can force raft log truncate on specific logstore. This will trigger on every logstore truncate always.